clab/inventory.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const NornirPlatformNameSchemaEnvVar = "CLAB_NORNIR_PLATFORM_NAME_SCHEMA"
 
-//go:embed inventory_ansible.go.tpl
+//go:embed assets/inventory_ansible.go.tpl
```

Comment:
all packages that had assets I just stuffed into an "assets" directory in their package to make it stupid obvious and to not have non ".go" files breaking up the flow of things in the main package